packages-content-model/roosterjs-content-model-types/lib/parameter/DOMHelper.ts

```diff
@@ -48,4 +48,10 @@ export interface DOMHelper {
      * @param name Name of the attribute
      */
     getDomAttribute(name: string): string | null;
+
+    /**
+     * Get DOM style of editor content DIV
+     * @param style Name of the style
+     */
+    getDomStyle(style: keyof CSSStyleDeclaration): string | number | object | undefined;
```

Comment:
Instead of returning string|number|..., can you try this:
```ts
getDomStyle<T extends keyof CSSStyleDeclaration>(style: T): CSSStyleDeclaration[T];
```

So we have a strong-typed return value

Reply:
I made the change but it looks like now all return values are of type string.

Reply:
hmm you are right. But I think it is by design. When you go to the declaration of CssStyleDeclaration, the type of its members are really string:

![image](https://github.com/microsoft/roosterjs/assets/23065085/283937f1-058f-406e-8e90-d2098dc037b4)

Reply:
And that also makes sense. Is there any example of CSS style value that is not string?

Reply:
The one I can think of related to the editor is z-index, which is a number.

Reply:
According to its definition, it is also string:

![image](https://github.com/microsoft/roosterjs/assets/23065085/2ddf0299-c0f9-4b14-97a6-b9afb18a6ccf)

Reply:
And according to https://www.w3schools.com/jsref/prop_style_zindex.asp, the javascript property of zIndex is also assigned as string:

![image](https://github.com/microsoft/roosterjs/assets/23065085/bf443248-aedf-4536-bfd3-4da2669406cc)

So, kind of per standard.

Reply:
Seeing as it is easy to convert string to number, I think it can stay this way.